common/tags/pom.xml

```diff
@@ -46,5 +46,24 @@
   <build>
     <outputDirectory>target/scala-${scala.binary.version}/classes</outputDirectory>
     <testOutputDirectory>target/scala-${scala.binary.version}/test-classes</testOutputDirectory>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <goals><goal>add-source</goal></goals>
+            <configuration>
+              <sources>
+                <source>src/main/java</source>
```

Comment:
The plugin already knows about src/test though. I think your problem is something else, like what's below or this: you have a problem with the new directory structure. They're now under a directory called "org.apache.spark.tags" instead of nested dirs "org/apache/spark/tags"